getting_hired/preparing_for_job_search/professional_networking.md

```diff
@@ -0,0 +1,82 @@
+* Scour local business publications for notices on local conferences and tech events.
+* Talk to friends and family to see who might be a software developer you could be introduced to.
+* Follow local tech companies on social media to learn about their in-person events.
+* Sign up for newsletters focused on content in your local tech scene.
+* Start your own Meetup group focused on a particular topic, e.g. Ruby or JavaScript.
+
+Your virtual networking options are much broader and there’s no shortage of platforms and communities to choose from. Consider the following to help you decide on where and how to focus your efforts:
+
+* [Join the Odin community](https://www.theodinproject.com/lessons/foundations-join-the-odin-community).
+* [LinkedIn](https://www.linkedin.com/) - your profile acts like your virtual resume and first impressions are important. Put some time into [optimizing your profile](https://www.samanthaming.com/blog/tips-to-optimize-your-linkedin-profile-for-developers/). Then start following companies you would like to work at or find interesting, and individuals you can learn from. Engage with their content with questions and comments and consider creating your own content about your developer journey. Connect with people who engage with you and aim to make these connections genuine by looking at someone’s profile and finding something in common to talk about.
+* [X](https://twitter.com/) - this is a great platform to keep a finger on the pulse of a given domain. You can follow companies and individuals, engage with their posts, and create your own content.
+* [Discord](https://discord.com/) - there’s likely a discord channel dedicated to a given language or framework where you can ask questions and get help. You can sign up for Discord and search for what you are interested in e.g., NodeJS, Ruby on Rails, and request to join.
+* Open Source projects - when you have the skillset and confidence to contribute to open-source projects, you will be working directly with project maintainers and contributors. It is a great way to connect with other developers while working towards a common good.
+
+Once you have built up your network a little, a great way to find out good information quickly is through [informational interviews](https://blog.get-merit.com/break-into-tech-with-informational-interviewing/). An informational interview is the formal name for a chat with someone who is in a position you want to be in, or are interested in learning about. You get to ask them questions, such as "What path did you take to get to this point?", "What do you do in a typical work day?", "What advice would you give to someone like me?", and "Tell me about the challenges you face in your role". The questions you ask should help you find out if a particular role or company is the right fit for you and if you need to make any adjustments in your strategy.
```

Comment:
One tiny addition that I think would be useful here is to express how we can leverage contacts we make in informational interviews into professional contacts.

Reply:
Ah right. Yes I was thinking we would recommend making a contact first then interviewing, but consitent with the resource provided, that would make sense.

Reply:
Done!